benchmark_joins/src/main.rs

```diff
@@ -24,10 +24,11 @@ fn main() {
 		right_table_names.push(rtn.to_owned());
 	}
 
-	// Parse left_block_size, right_block_size, and join_algo
+	// Parse left_block_size, right_block_size, join_algo, and num_trials
 	let left_block_size: usize = args.get(4).unwrap().parse().unwrap();
 	let right_block_size: usize = args.get(5).unwrap().parse().unwrap();
 	let raw_join_algo: &str = args.get(6).unwrap();
+	let num_trials: i8 = args.get(7).unwrap().parse().unwrap();
```

Comment:
any reason this is i8? Max value of i8 is 127 which is scary imo. Wouldn't hurt to have it be i32 and have no chance of overflow